### DISCUSSION TITLE
[gatsby-theme-cara] Theme-UI Not compatible with Gatsby v4?

### DISCUSSION THREAD

**Discussion by ebarnette** [LekoArts/gatsby-themes#832] [Questions] [answered]
![image](https://user-images.githubusercontent.com/14867137/155860812-9fc13cab-a11a-4b2d-a465-9415736e7df1.png)

When I compile the base project, after cloning the template, I see these warnings.
@LekoArts do you know anything about Theme-UI not working in this scenario? I'm having a really hard time trying to change any colors at all on the website. I've shadowed gatsby-plugin-theme-ui correctly but nothing is changing when I change values under style > colors.

File structure:

```
.
├── @lekoarts
│   └── gatsby-theme-cara
│       ├── components
│       │   └── component files
├── gatsby-plugin-theme-ui
│   └── index.js
└── pages
    └── 404.jsx
```

I ended up having to hardcode the background in Layout.tsx by specifying it in the html tag (since it gets overridden by the theme-ui defaults anywhere else):

```
html: {
          fontSize: `18px`,
          WebkitTextSizeAdjust: `100%`,
          backgroundColor: `#FAFAF9`,
        },
```

I was able to change the background color but can't get any text colors to change. Here's my site: https://www.munchables.io/ (it's a WIP)

**Comment by ruiG** [comment]
I have the same exact problem with the Jodie theme.

**Comment by LekoArts** [comment, marked as answer]
The plugin is compatible with Gatsby 4, it's only a matter of correct `peerDeps` set. I'll want to update to Theme UI 0.14 soonish (https://github.com/LekoArts/gatsby-themes/projects/3#card-75896344) which will mitigate this.

But it all works, it's just a npm warning
